app/helpers/search_bar_helper.rb

```diff
@@ -0,0 +1,21 @@
+module SearchBarHelper
+  def mount_search_bar(id, use_shortcuts = true, use_bookmarks = true)
```

Comment:
I'd prefer to either use hash as options or better, keyword arguments, basically that only means to change changing ` use_shortcuts = true, use_bookmarks = true` to ` use_shortcuts: true, use_bookmarks: true`. And then when you call the method, the key name must be specified. The difference/benefit is that when we add more options, it does not change the method prototype too much, the arguments position does not matter.

And since this is a helper that will be used from more places, I'd prefer to keep it easily extendable

Another thing that I just realized, this assumes the controller can determine the url based on its name. While it might not be blocking this PR, it would be great to be able to pass the url as another param (might be a nice little enhancement for other who wants to get hands dirty). What's more important I think though is the params[:search] which is hardcoded now to searchQuery. On other pages, I might want to link it to different parameter.